crt_portal/cts_forms/templates/forms/complaint_view/show/index.html

```diff
@@ -95,11 +95,4 @@ <h2 class="margin-right-205 margin-top-0 margin-bottom-0 backend-blue">
 <script src="{% static 'js/edit_details.js' %}"></script>
 <script src="{% static 'js/modal.js' %}"></script>
 <script src="{% static 'js/form_letter.js' %}"></script>
-<script type="text/javascript" src="{% static 'js/accessible-autocomplete.min.js' %}"></script>
-<script nonce="{{request.csp_nonce}}">
-  accessibleAutocomplete.enhanceSelectElement({
-  defaultValue: '',
-  selectElement: document.querySelector('#id_assigned_to')
-});
-</script>
```

Comment:
I believe we want to keep this -- specifically for the individual view page. Otherwise it looks like:

![2020-08-17_13-20](https://user-images.githubusercontent.com/3013175/90440664-78813b80-e08c-11ea-8a8f-0bd0c527a3e1.png)